### ISSUE TITLE
Could not decode image: 'logo-white-bgblue.svg'

### ISSUE THREAD

**Issue by exiadb** [CanopyTax/single-spa-inspector#15]
https://chrome.google.com/webstore/detail/single-spa-inspector/emldbibkihanfiaiaghebffnbahjcgcp

Cannot install the extension.

Version 72.0.3626.119 (Official Build) (64-bit)

Mac os 10.14.3 (18D109)

**Comment by frehner** [comment]
ah looks like it's because chrome/chromium can't take svgs as the icon, whereas Firefox can. 

the solution appears to be to just make a .png of the logo and change the manifest to refer to it instead of the svg.

**Comment by riccardomessineo** [comment]
Please build a new release!
The bug is preventing the installation of the plugin... :(

**Comment by frehner** [comment]
I believe we submitted it already, but the chrome store takes days to approve and then roll out the update. Hopefully it's out there soon!

**Comment by frehner** [comment]
Just as a status update: looks like we're still waiting on google to review the app and publish the latest update. I've seen it take up to 5 days before, so we're on day 3 now I think :/

**Comment by frehner** [comment]
Looks like the chrome store finally published the update, and I was able to install it. Let me know if you have any other issues!

**Comment by exiadb** [comment]
That's all working now.

Thanks for publishing the new version!

**Comment by hieuxuanta** [comment]
I converted all images to .png and set it to tab background, NTP bg,....etc... but it still pop up the error "Could not decode image 'myImage.png' ". Could someone give me another solutions? Tks!

**Comment by frehner** [comment]
>I converted all images to .png and set it to tab background, NTP bg,....etc... but it still pop up the error "Could not decode image 'myImage.png' ". Could someone give me another solutions? Tks!

I think you're in the wrong repo?

Could create a new issue and explain the context of the problem that you've run into?
